libs/coin-modules/coin-sui/src/network/sdk.ts

```diff
@@ -260,6 +278,83 @@ function transactionToOp(address: string, transaction: SuiTransactionBlockRespon
   };
 }
 
+/**
+ * Convert a SUI RPC checkpoint info to a {@link BlockInfo}.
+ *
+ * @param checkpoint SUI RPC checkpoint info
+ */
+export function suiCheckpointToBlockInfo(checkpoint: Checkpoint): BlockInfo {
```

Comment:
optional wdyt (same for other function)
```suggestion
export function toBlockInfo(checkpoint: Checkpoint): BlockInfo {
```

Reply:
ok, changed it all